src/airodump-ng/airodump-ng.c

```diff
@@ -842,6 +843,10 @@ static const char usage[] =
 	"      --netmask <netmask>   : Filter APs by mask\n"
 	"      --bssid     <bssid>   : Filter APs by BSSID,\n"
 	"                              you can pass multiple --bssid options\n"
+	"      --exclude-bssid\n"
+	"                  <bssid>   : Similar to --bssid but excludes the AP,\n"
+	"                              you can pass multiple --exclude-bssid "
+	"options\n"
```

Comment:
Please fix Indentation

Reply:
This is what it looks like:

```
$ ./airodump-ng --help
...
      --write-interval
                  <seconds> : Output file(s) write interval in seconds
      --background <enable> : Override background detection.

  Filter options:
      --encrypt   <suite>   : Filter APs by cipher suite,
                              you can pass multiple --encrypt options
      --netmask <netmask>   : Filter APs by mask
      --bssid     <bssid>   : Filter APs by BSSID,
                              you can pass multiple --bssid options
      --exclude-bssid
                  <bssid>   : Similar to --bssid but excludes the AP,
                              you can pass multiple --exclude-bssid options
      --essid     <essid>   : Filter APs by ESSID,
                              you can pass multiple --essid options
...
```

Writing `--exclude-bssid <bssid>` in one line would be too long so it would not fit, thats why I split it similarly to `--write-interval`. If you meant shifting it so there will be only 1 space between `<bssid>` and `:`, I updated it:

```
$ ./airodump-ng --help
...
      --exclude-bssid
                    <bssid> : Similar to --bssid but excludes the AP,
                              you can pass multiple --exclude-bssid options
...
```